imap_processing/idex/idex_l1a.py

```diff
@@ -131,9 +131,9 @@ def __init__(self, packet_file: Union[str, Path], data_version: str) -> None:
 
         dust_events = {}
         for packet in decom_packet_list:
-            if "IDX__SCI0TYPE" in packet.data:
-                scitype = packet.data["IDX__SCI0TYPE"].raw_value
-                event_number = packet.data["IDX__SCI0EVTNUM"].derived_value
+            if "IDX__SCI0TYPE" in packet:
+                scitype = packet["IDX__SCI0TYPE"].raw_value
```

Comment:
I'm guessing you can remove some of these `raw_value`s as well here.
```suggestion
                scitype = packet["IDX__SCI0TYPE"]
```

Reply:
I see your point but I don't want to assume and like to leave that to IDEX team to decide.

Reply:
I think you should be opinionated here and do things the "right" way because this was only done the other way due to the old space_packet_parser way of handling things. This is the new way and now that you are doing the update you should (IMO) update these for all the teams. (Otherwise it will never be done later)